packages/loopring_v3/contracts/amm/AmplifiedAmmController.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.7.0;
+
+import "./IAmmController.sol";
+import "../amm/LoopringAmmPool.sol";
+import "../lib/Claimable.sol";
+import "../lib/MathUint.sol";
+import "../lib/MathUint96.sol";
+import "../thirdparty/SafeCast.sol";
+
+
+/// @author Brecht Devos - <brecht@loopring.org>
+contract AmplifiedAmmController is IAmmController, Claimable
+{
+    using MathUint          for uint;
+    using MathUint96        for uint96;
+    using SafeCast          for uint;
+
+    uint public constant AMPLIFICATION_FACTOR_BASE = (10 ** 18);
```

Comment:
No, that's not necessary. It could be any value as this doesn't impact the protocol in any way. The protocol only sees the virtual balances (which are also base/decimals/... agnostic). This value is only important in the `AmplifiedAmmController` itself for how the AF values are stored and used.

Reply:
>This value is only important in the AmplifiedAmmController itself for how the AF values are stored and used.
So if we choose a default value, `AmplifiedAmmController` wont' get AF == 1, right?

Reply:
I see: the amplification factors are stored in the mapping inside `AmplifiedAmmController`.